crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1129,6 +1092,11 @@ impl<T: R2D2Connection> IndexerStore for PgIndexerStore<T> {
                 }
             }
         }
+        let object_deletions = object_deletions
```

Comment:
Why not do this `into` on line 1091 already while pushing into `object_deletions`?